presto-native-execution/presto_cpp/main/tests/PrestoExchangeSourceTest.cpp

```diff
@@ -429,7 +430,7 @@ TEST_P(PrestoExchangeSourceTestSuite, basic) {
   auto producerAddress = serverWrapper.start().get();
   auto producerUri = makeProducerUri(producerAddress, useHttps);
 
-  auto queue = std::make_shared<exec::ExchangeQueue>(1 << 20);
+  auto queue = std::make_shared<exec::ExchangeQueue>();
```

Comment:
this constructor parameter was removed in 

https://github.com/facebookincubator/velox/pull/5884

Cc: @mbasmanova